src/views/htmlcontent/src/js/slick.dragrowselector.js

```diff
@@ -116,10 +116,16 @@
 
                 if (e.shiftKey) {
                     var last = _ranges.pop();
+
+                    var toCell = Math.max(newRangeColumn, last.toCell);
+                    if(last.fromCell <= newRangeColumn) {
```

Comment:
could you add a ' ' between the 'f' and '(' in conditional statements to match existing coding format of the file.